geo.txt

```diff
@@ -56,3 +56,1703 @@ Phys 91(11):957-962
+# Bond lengths: r0 (angstroms)
+# Source paper: Bittner DM, Zaleski DP, Tew DP, Walker NR, Legon AC (2016) Highly unsaturated platinum and palladium
+carbenes PtC3 and PdC3 isolated and characterized in the gas phase. Angew Chem 128(11):3832-3835; Angew
+Chem Int Ed Engl 55(11):3768-3771
+# Date first added: 2023-07-11 (by Sichao Xiong)
+
+Pd   = C(1) 1.79898(4)
+C(1) = C(2) 1.3009
+C(2) = C(3) 1.2789
+
+# C(1) = C(2) 1.3009 and C(2) = C(3) 1.2789 fixed to the value calculated at the level CCSD(T)_ae/aug-cc-pwCV5Z-PP(for Pd) and corrected for r0 ‒ re
+difference assumed to that for PtC3. 
+
+NAME = C3Pt
+# Point Group: Coov 
+# Shape: Linear
+# Comment: The molecule was found to be linear or quasilinear.
+# Bond lengths: r0 (angstroms)
+# Source paper: Bittner DM, Zaleski DP, Tew DP, Walker NR, Legon AC (2016) Highly unsaturated platinum and palladium
+carbenes PtC3 and PdC3 isolated and characterized in the gas phase. Angew Chem 128(11):3832-3835; Angew
+Chem Int Ed Engl 55(11):3768-3771
+# Date first added: 2023-07-11 (by Sichao Xiong)
+
+Pt   = C(1) 1.7315(14)
+C(1) = C(2) 1.2993(19)
+C(2) = C(3) 1.2759(11)
+
+NAME = C4HNO 2012
+# Point Group: Coov 
+# Shape: Linear
+# Bond lengths: r0 (angstroms)
+# Source paper: Kang L, Novick SE (2012) The microwave spectra of the weakly bound complex between carbon monoxide and
+cyanoacetylene, OC H–C≡C–C≡N. J Mol Spectrosc 276-277:10-13
+# Date first added: 2023-07-11 (by Sichao Xiong)
+
+C(4)...H        2.65315(27)
+C(4)...D        2.64962(27)
+OC4    HC1C2C3N 6.2031
+
+NAME = C4HNO2 2017
+# Point Group: Coov 
+# Shape: Linear --- Linear
+# Bond lengths: r0     (angstroms)
+# Bond angles:  theta0 (degrees)
+# Source paper: Kang L, Davis P, Dorell I, Li K, Oncer O, Wang L, Novick SE, Kukolich SG (2017) Rotational spectrum and
+structure of the T-shaped cyanoacetylene carbon dioxide complex, HCCCN⋅⋅⋅CO2. J Mol Spectrosc 342(6):62-72
+# Date first added: 2023-07-11 (by Sichao Xiong)
+
+OCO NCCCH 4.8311(3)
+ϕ         10(1)
+φ         5.4(5)
+
+# ϕ: Angle of torsional oscillation about the a axis, defined as the angle between this axis and the HCCCN axis
+# φ: Angle of torsional oscillation about an axis parallel to b one (bꞌ), defined as the angle between bꞌ and the
+O=C=O axis
+
+NAME = C6H2N2 2016
+# Point Group: Coov 
+# Shape: Linear --- Linear
+# Bond lengths: r0     (angstroms)
+# Bond angles:  theta0 (degrees)
+# Source paper: Kang L, Davis P, Dorell I, Li K, Daly A, Novick SE, Kukolich SG (2016) Rotational spectra and nitrogen
+nuclear quadrupole coupling for the cyanoacetylene dimer: H-C≡C-C≡N⋅⋅⋅H-C≡C-C≡N. J Mol Spectrosc
+321(3):5-12
+# Date first added: 2023-07-11 (by Sichao Xiong)
+
+# Assuming linear model
+
+N(1)...H(2) 2.2489(3)
+
+# Using rigid precession model.
+
+N(1)...H(2)   2.3415
+HCCCN(1) H(2)CCCN 7.0603
+ϕ1            13.0
+ϕ2            8.7
+
+# ϕ1: Angle between the a axis and the axis of the HCCCN(1) subunit.
+# ϕ2: Angle between the a axis and the axis of the other HCCCN subunit
```

Comment:
A Coov molecule wouldn't have an angle though, so this is similar to the previous case in which we had two different point groups for the same molecule, but  the book didn't list them both equally at the beginning.